src/pages/claim-details/claim-details.tsx

```diff
@@ -158,7 +158,7 @@ function ClaimSummary(props: ClaimSummaryProps) {
       </div>
       <div className={styles.detailsItem}>
         <p className={globalStyles.bold}>Claim Created</p>
-        <p className={globalStyles.secondaryColor}>{format(claim.timestamp, 'do MMMM yyyy hh:mm')}</p>
+        <p className={globalStyles.secondaryColor}>{format(claim.timestamp, 'do MMMM yyyy HH:mm')}</p>
```

Comment:
Can we maybe extract these to a constants and reuse? We are inlining these formats
<img width="470" alt="image" src="https://user-images.githubusercontent.com/22679154/194745256-2fc9840b-d078-402a-a2fb-650e9b0c3bb5.png">

Reply:
One pain of doing this is that there are no natural names for these formats. E.g. here are some (probably bad) names that I came up with:
```ts
export const FULL_DATE_FORMAT = 'do MMMM yyyy';
export const COMPACT_DATE_FORMAT = 'dd MMM yyyy';
```
Now say we want to use `do MMM yyyy`, what would we call it?

But ja, I'm not too fussed, did you have specific naming in mind?

Reply:
Merging so long. Depending on the outcome of this discussion, we can create a new task for it.